aboutus.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="DRRP Website">
+
+		<title>Transition to Employment</title>
+
+		<!--- google fonts -->
+		<link href="https://fonts.googleapis.com/css?family=Fjalla+One" rel="stylesheet"> 
+
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+		
+	</head>
+	
+	<body>
+		<div class="container">
+		<header class="clearfix">
+			<div class="logo">
+				<a href="index.html"><img src="images/logo.png" alt="DRRP Transition to Employment logo"></a>
+			</div>
+				<nav>
+					<ul>
+						<li><a href="aboutus.html">ABOUT US</a>
+							<ul>
+								<li><a href="studies.html">Studies</a></li>
+								<li><a href="partners.html">Partners</a></li>
+								<li><a href="advisors.html">Advisors</a></li>
+							</ul>
+						</li>
+						<li><a href="toolkit.html">TOOLKIT</a>
+							<ul class="navalign">
+								<li><a href="collaboration.html">Collaboration</a></li>
+								<li><a href="employment.html">Employment</a></li>
+								<li><a href="intervention.html">Intervention</a></li>
+								<li><a href="skilledprofessional.html">Skilled Profesional</a></li>
+							</ul>
+						</li>
+						<li><a href="contactus.html">CONTACT</a></li>
+					</ul>
+				</nav>
+		</header>
+
+		<section>
+			<div class="mainphoto skilled">
+				<h1 class="homepageheader">ABOUT US</h1>
+			</div>
+			&nbsp;
+			<div>
+				<h1>Center on Transition to Employment for Youth with Disabilities</h1>
+					<h3>The purpose of the Center on Transition to Employment for Youth with Disabilities is to identify, research, and widely disseminate evidence-based practices that contribute to successful employment for youth with disabilities making the transition from school to adult employment. Through six discrete studies, the Center is designed to research and spotlight the strategies and circumstances that produce optimal employment and career achievement for these youth, who continue to transition from school to adult life with much lower prospects for employment success than their non-disabled peers. Thus, the Center will provide a comprehensive and contemporary view of the strategies and models of school to work transition for youth with disabilities in our country.</h3>
```

Comment:
This seems like a lot of copy of header tag. Maybe make it a `<p>` with a special class that makes the text bigger if that's the desired effect?